site/app/controllers/admin/LateController.php

```diff
@@ -25,13 +25,22 @@ public function run() {
      * @return Response
      */
     public function viewLateDays() {
+<<<<<<< HEAD
```

Comment:
merge conflict isn't resolved yet

Reply:
Looks like I did not commit properly. Please check one more time. :)